.api/v1.4.0/PowerBI-visuals.d.ts

```diff
@@ -893,6 +933,7 @@ declare module powerbi {
         image?: ImageTypeDescriptor;
         paragraphs?: ParagraphsTypeDescriptor;
         geoJson?: GeoJsonTypeDescriptor;
+        queryTransform?: QueryTransformTypeDescriptor;
```

Comment:
It'd be better to stop keeping API in the repository.
We might [install API automatically](https://github.com/Microsoft/PowerBI-visuals-ChicletSlicer/blob/master/package.json#L14).